assets/prototype/application/utilities/money/formatAmount.ts

```diff
@@ -0,0 +1,16 @@
+import parsedAmount from './parsedAmount';
+
+/**
+ * returns a function that when supplied a value for the number of decimal places used by a currency,
+ * returns a second function that can be passed an amount which will then be appropriately formatted
+ *
+ * @param {number} decimalPlaces
+ * @return {Function}
+ */
+const formatAmount = (decimalPlaces: number): ((amount: number | string) => string) => (
```

Comment:
Maybe return type can be extracted somehow?

Reply:
Maybe return can be used from `MoneyDisplay.formatAmount` from `useMoneyDisplay.ts`?

Reply:
do you mean into a named type/interface?

something like:

```ts
const formatAmount = (decimalPlaces: number): AmountFormatter => (
```

Reply:
Yes, the example from above is more grokkable.

Reply:
ok I can do that... expect another review request in a bit